lib/copy_peste/core/module_mng.rb

```diff
@@ -0,0 +1,51 @@
+require_relative '../modules.rb'
```

Comment:
Do you really need to require those files here? I think they are already required at a higher level.
If removing doesn't work, maybe try to change `require`s order in the upper files.